azure-cs-botservice/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+using Pulumi;
+using Pulumi.Azure.AppService;
+using Pulumi.Azure.AppService.Inputs;
+using Pulumi.Azure.Core;
+using Pulumi.Azure.Storage;
+using Pulumi.AzureAD;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+class Program
+{
+    static Task<int> Main(string[] args)
+    {
+        return Deployment.RunAsync(() =>
+        {
+            var config = new Config();
+            var botName = config.Require("botName");
+            var botSecret = config.GetSecret("botSecret");
+
+            var resourceGroup = new ResourceGroup("botservice-rg", new ResourceGroupArgs
+            {
+            });
```

Comment:
No need to pass empty args here: you can just omit the second parameter of `new ResouceGroup()`

Reply:
Much cleaner :)